src/Selenium2Library/keywords/element.py

```diff
@@ -217,6 +217,23 @@ def element_should_be_enabled(self, locator):
             raise AssertionError("Element '%s' is disabled." % (locator))
 
     @keyword
+    def element_should_be_focused(self, locator):
+        """Verifies that element identified with `locator` is focused.
+
+        Key attributes for arbitrary elements are `id` and `name`. See
+        `introduction` for details about locating elements.
+
+        New in SeleniumLibrary 2.0.1.
+        """
+        element = self.find_element(locator)
+        try:
+            focused = self.browser.execute_script('return document.activeElement;')
+        except (IOError, RuntimeError) as err:
+            raise AssertionError("Internal Error: '%s'" % (err))
```

Comment:
Drivers inherits the remote, see the documentation example for the `Chrome`: http://selenium-python.readthedocs.io/api.html#module-selenium.webdriver.chrome.webdriver

Also this works fine: 
```
from selenium import webdriver
from selenium.webdriver.remote.webelement import WebElement

driver = webdriver.Chrome()
driver.get('http://robotframework.org/')
focus = driver.switch_to.active_element
print isinstance(focus, WebElement)
print focus
driver.quit()
```
Please change the code to use the `driver.switch_to.active_element`